assets/js/src/bulk-delete.js

```diff
@@ -168,11 +168,19 @@ jQuery(document).ready(function () {
 		}
 	};
 
-    BulkWP.validateUserMeta = function() {
-        if (jQuery('#smbd_u_meta_value').val() !== '') {
-            return true;
-        } else {
-            return false;
-        }
-    };
+	BulkWP.validateUserMeta = function() {
+		if (jQuery('#smbd_u_meta_value').val() !== '') {
+			return true;
+		} else {
+			return false;
+		}
+	};
+
+	BulkWP.validateWordCount = function() {
+		if ( jQuery('#smbd_content' ).val() !== '') {
```

Comment:
This function should be added in the add-on, not in core.

Also this can be simplified as just `return jQuery('#smbd_content' ).val() !== ''`

Reply:
Fixed both.

Reply:
The JS has been moved to the add-on and the corresponding PR is [here](https://github.com/sudar/bulk-delete-posts-by-word-count/compare/fix/review-comments?expand=1).